node/src/Commands.ts

```diff
@@ -2803,6 +2803,66 @@ export function createMove(
     return createCommand(RequestType.Move, [key, dbIndex.toString()]);
 }
 
+/**
+ * @internal
+ */
+export function createDump(key: GlideString): command_request.Command {
+    return createCommand(RequestType.Dump, [key]);
+}
+
+/** Optional arguments for `RESTORE` command. */
+export type RestoreOptions = {
+    /**
+     * Set to `true` to replace the key if it exists.
+     */
+    replace?: boolean;
+    /**
+     * Set to `true` to specify that `ttl` argument of {@link BaseClient.restore} represents
+     * an absolute Unix timestamp (in milliseconds).
+     */
+    absttl?: boolean;
+    /**
+     * Set the `IDLETIME` option with object idletime to the given key.
```

Comment:
```suggestion
     * Set the `IDLETIME` option with object idletime to the given key.
     * Cannot be specified together with {@link frequency}.
```
or something like this.

Reply:
I have put it at the top.